docs/source/get_started/installing.rst

```diff
@@ -28,11 +28,16 @@ Install and Configure Dependencies
 
 			MySQL is going to ask you for a root password. You have to input it twice. Remember it because we'll use this later.
 
-	#. Change the ownership of the tomcat7 directories
+	#. Change memory allocation for catalina when it starts and the ownership of the tomcat7 directories
+
+		..note::
+
+		#Note, you can change the memory allocation to anything, but we recommend at least 512m.
 
 		.. code-block:: bash
 
 			sudo service tomcat7 stop
+			sudo sed -i '2iCATALINA_OPTS="-Xms512m -Xmx512m"' /usr/share/tomcat7/bin/catalina.sh
```

Comment:
I think it's best to not change catalina.sh
you can either set the CATALINA_OPTS environment variable in your .profile or .bashrc file (in you run tomcat as yourself) or create a setenv.sh file alongside catalina.sh and it will be picked up automatically.